api/__init__.py

```diff
@@ -0,0 +1,92 @@
+"""Backend api server that is used in the MOH project.
+
+A simple flask backend that will maintaine a live queue of current students in office hours and keep
+track of current student status in the within the queue
+by making request updating the `vist` table within the external postgrest database.
+"""
+
+# TODO: Make the doc string sound better
+
+
+import datetime
+from flask import Flask, request
+
+
+def create_app():
+    """Create and return flask api server
+
+    This funciton is used to setup the flask api server loading all it's depended modulars
+    """
+    app = Flask(__name__)
+
+    @app.route("/login", methods=["POST"])
+    def login():
+        """Checks if the current user have the right credentials to login
```

Comment:
"Log in" is a verb; "login" is a noun.

```suggestion
        """Checks if the current user has the right credentials to log in
```

Though, I still don't think this description is accurate for the endpoint. There is probably no "current user" at the point this request is made.